custom/scripts/eks/README.md

```diff
@@ -0,0 +1,119 @@
+# Setup EKS And Configure Kafka
+A terraform script to create a managed kubernetes cluster on AWS EKS with the spot Instance.This script also gives you an option to setup kafka along with prometheus and kafka-provisioned grafana and also litmus portal.
+## Prerequisites
+Make sure You have installed all of the following prerequisites on your system:
+* Kubectl - [Download & Install kubectl](https://kubernetes.io/docs/tasks/tools/) 
+* aws-cli - [Download & Install aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-windows.html) and also configure the awscli.
+* Terraform - [Download & Install terraform](https://www.terraform.io/downloads.html)
+
+## Step 1: 
+  Clone the repo
+  ```
+  git clone https://github.com/Adarshkumar14/kafka-eks-infra-setup.git
+  ```
+## Step 2:
+### Variables
+You can change the value of variable in the variable.tf
+ <table>
+    <tr>
+      <th> Name </th>
+      <th> Description</th>
+      <th> Example </th>
+    </tr>
+    <tr>
+      <td> <code>vpc_name</code> </td>
+      <td> The name of VPC</td>
+        <td><code>eks-vpc</code> </td>
+    </tr>
+    <tr>
+      <td><code>vpc_cidr</code></td>
+      <td> Subnet CIDR</td>
+      <td><code>10.0.0.0/16</code></td>
+    </tr>
+    <tr>
+      <td><code>private_subnets_cidr</code></td>
+      <td>Private Subnet CIDR</td>
+      <td><code>[ "10.0.1.0/24" , "10.0.2.0/24", "10.0.3.0/24",]</code></td>
+  </tr>
+  <tr>
+    <td><code>public_subnets_cidr</code></td>
+    <td>Public Subnet CIDR</td>
+    <td><code>[ "10.0.101.0/24" , "10.0.102.0/24", "10.0.103.0/24",]</code></td>
+  </tr>
+  <tr>
+    <td><code>azs</code></td>
+    <td>AWS Availability Zones</td>
+    <td> <code>[ "ap-south-1a" , "ap-south-1b", "ap-south-1c"]</code></td>
+  </tr>
+  <tr>
+    <td><code>cluster_name</code></td>
+    <td>The name of your EKS Cluster</td>
+    <td><code>eks-adarsh</code></td>
+  </tr>
+  <tr>
+    <td><code>k8s_version</code></td>
+    <td>The desired K8s version to launch</td>
+    <td><code>1.17</code></td>
+  </tr>
+  <tr>
+    <td><code>node_instance_type</code></td>
+    <td>Worker Node instance type</td>
+    <td><code>t3.xlarge</code></td>
+  </tr>
+  <tr>
+    <td><code>desired_nodes</code></td>
+    <td> Autoscaling Desired node capacity</td>
+    <td><code>3</code></td>
+  </tr>
+  <tr>
+    <td><code>max_nodes</code></td>
+    <td>Autoscaling Maximum node capacity</td>
+    <td><code>3</code></td>
+  </tr>
+  <tr>
+    <td><code>spot_price</code></td>
+    <td> Max spot Price</td>
+    <td><code>0.10</code></td>
+  </tr>
+  <tr>
+    <td><code>configure_kafka</code></td>
+    <td>To configure kafka enter true else false </td>
+    <td> <code>true</code> or <code>false</code> </td>
+  </tr>
+  </table>
+  
+##  Step 3:
+   Provide the provider in providers.tf
+   ```
+   provider "aws" {
+    region = "ap-south-1"
+    profile = "default"
+}
+```
```

Comment:
Can we also add something about tags, the user can change the tags according to their preference. Otherwise, the default tags will be used.

Reply:
@Adarshkumar14 Can you add something here about tags, how users can change them. As it needs to be changed from main.tf. cc: @uditgaurav